dashboard/test/models/user_test.rb

```diff
@@ -5473,6 +5473,70 @@ def hide_scripts_in_sections(section1, section2)
     assert_equal new_us_state, student.reload.us_state
   end
 
+  describe 'strict password requirements by country' do
+    let(:strict_country) {User::PASSWORD_STRICT_COUNTRIES.first}
+
+    before do
+      allow(DCDO).to receive(:get).with('strict-password-country', false).and_return(true)
+      allow(DCDO).to receive(:get).with('migration_service_enabled', false).and_return(false)
+    end
+
+    describe 'when creating a teacher in a strict country' do
+      context 'when using a short password' do
+        let(:user) {build(:teacher, country_code: strict_country, password: 'short', password_confirmation: 'short')}
+        it 'rejects passwords' do
+          _(user).wont_be :valid?
+        end
+
+        it 'adds an error message' do
+          user.valid?
+          _(user.errors[:password]).must_include 'is too short (minimum is 14 characters)'
+        end
+      end
+      context 'when using a short password' do
```

Comment:
I think you missed this
```suggestion
      context 'when using a long password' do
```